src/Microsoft.Health.Fhir.Api/Features/Headers/OperationHeaders.cs

```diff
@@ -0,0 +1,35 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using EnsureThat;
+using Microsoft.Health.Fhir.Api.Features.ActionResults;
+using Microsoft.Health.Fhir.Core.Features.Routing;
+using Microsoft.Net.Http.Headers;
+
+namespace Microsoft.Health.Fhir.Api.Features.Headers
+{
+    public static class OperationHeaders
```

Comment:
Can you renamed this to be named OperationResultExtensions and rename FhirHeaders to be FhirResultExtensions too?